src/Resources/config/database_tools.xml

```diff
@@ -7,6 +7,7 @@
     <services>
         <service id="liip_functional_test.services.fixtures_loader_factory" class="Liip\FunctionalTestBundle\Services\FixturesLoaderFactory" public="true">
             <argument type="service" id="service_container" />
+            <argument type="service" id="doctrine.fixtures.loader" />
```

Comment:
Before adding this hard dependency, you have to either require it in the `composer.json`, or make this optional (and add approriate check later in the code).

I heavily lean on the latter, since requiring an entire Doctrine bundle where it could not be wanted is too hard of a requirement.

Reply:
Thanks for you input. This indeed should be handled differently. 

Should I mimic the behavior of the `\Doctrine\Bundle\FixturesBundle\Loader\SymfonyFixturesLoader` in this bundle directly? 
eg. I get all service tagged with "doctrine.orm.fixtures" and return their instance instead of waiting to create them?